attributioncode/validator.go

```diff
@@ -5,18 +5,30 @@ import (
 	"crypto/sha256"
 	"encoding/base64"
 	"encoding/hex"
+	"fmt"
 	"net/url"
 	"time"
 
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 )
 
+const maxUnescapedCodeLen = 600
```

Comment:
Also just noting and probably don't need to change anything here either -- Firefox does a length check against the escaped code length while the check here is against the unescaped length.

Reply:
At least, I think it should match what fx has, with a comment pointing to it.

https://searchfox.org/mozilla-central/rev/a92ed79b0bc746159fc31af1586adbfa9e45e264/browser/components/attribution/AttributionCode.jsm#24